docs/sql-data-sources-avro.md

```diff
@@ -234,7 +234,9 @@ Data source options of Avro can be set via:
           When reading Avro, this option can be set to an evolved schema, which is compatible but different with
           the actual Avro schema. The deserialization schema will be consistent with the evolved schema.
           For example, if we set an evolved schema containing one additional column with a default value,
-          the reading result in Spark will contain the new column too.
+          the reading result in Spark will contain the new column too. Note that when using this option with 
+          <code>from_avro</code>, you still need to pass the actual schema as a parameter to the function. 
+          Otherwise, the behavior is undefined: it may fail or return arbitrary result.
```

Comment:
The actual schema is required by the from_avro API. Thus, the "otherwise" case won't happen.
I suggest we explain that the jsonSchema is for the actual input schema, while the "avroSchema" is for the output schema

Reply:
Make sense. Removed the "otherwise" clause.
I think emphasizing on the difference between "actual" and "evolved" should be enough for this case? Further mentioning input vs output seems a bit wordy for me... @gengliangwang